CHANGELOG.md

```diff
@@ -1,6 +1,48 @@
 # Changelog
 
-## [Unreleased](https://github.com/MarquezProject/marquez/compare/0.47.0...HEAD)
+## [Unreleased](https://github.com/MarquezProject/marquez/compare/0.48.0...HEAD)
+
+## [0.48.0](https://github.com/MarquezProject/marquez/compare/0.47.0...0.48.0) - 2024-07-30
+
+### Added
+* API: add endpoint method and path to metrics name [`#2850`](https://github.com/MarquezProject/marquez/pull/2850) [@JDarDagran](https://github.com/JDarDagran)  
```

Comment:
It's actually opposite - both are added as labels together with SQL object and method.

Reply:
@JDarDagran changed this, wdyt?

Reply:
uhm, I still can see `add endpoint method and path to metrics name` 
I'd rather say - add new labelled DB metrics or something like that.